mantle/kola/tests/misc/network.go

```diff
@@ -84,30 +94,33 @@ NextProcess:
 			c.Fatalf("unexpected number of parts on line: %q in output %q", line, output)
```

Comment:
Is the above logic still applicable? i.e. is udp's `state` field still optional such that we could end up with 6 columns versus 7? If not then let's nuke that logic.

Reply:
You're right. The `netstat` manpage explicitly mentions it is optional, whereas the `ss` manpage does not. So I'm assuming that `ss` output always has the `state` field.